apps/explorer/src/pages/AddressInfoPage/AssetList.tsx

```diff
@@ -50,30 +51,51 @@ const AssetList = ({ addressHash, addressBalance, limit, className }: AssetListP
   const {
     fungibleTokens: fungibleTokensMetadata,
     nfts: nftsMetadata,
-    unknown: unknownAssetsIds,
-    isLoading: assetsMetadataLoading
+    unknown: unknownAssetsIds
   } = useAssetsMetadata(assetIds)
 
-  const isLoading = assetsLoading || assetsMetadataLoading
+  const tokensWithAvailablePrice = useTokensWithAvailablePrice()
+  const tokensPrices = useTokensPrices([
+    ALPH.symbol,
+    ...fungibleTokensMetadata.flatMap((t) => (tokensWithAvailablePrice.includes(t.symbol) && t.symbol) || [])
```

Comment:
Ah that's an interesting one yes. I did this because the other way around I get `Type 'boolean' is not assignable to type 'string'.` :) But I can use a simple if instead.

Reply:
I actually want to return the symbol here, but only if I know it's one of the token for which we have a price. If I'm not mistaken, t.symbol is always defined here.